rules/windows/process_creation/proc_creation_win_curl_insecure_connection.yml

```diff
@@ -0,0 +1,24 @@
+title: Insecure Transfer Via Curl.EXE
+id: cb9cc1d1-e84e-4bdc-b7ad-c31b1b7908ec
+status: experimental
+description: Detects execution of "curl.exe" with the "--insecure" flag.
+references:
+    - Internal Research
+author: X__Junior (Nextron Systems)
+date: 2023/06/30
+tags:
+    - attack.execution
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_img:
+        - Image|endswith: '\curl.exe'
+        - OriginalFileName: 'curl.exe'
+    selection_cli:
+        - CommandLine|re: '\s-k\s'
```

Comment:
why specifically using `\s` instead of ` `?

Sadly I am unsure about the level. Low would be too low. Medium could be too high. So probably should be medium.

Reply:
Yeah, medium is the best we can give it for now. The usage of "\s" is to ensure consistency across other rules and to also take into consideration multiple spaces for backends that do not handle that well since there is no "contains"

Reply:
I don't get it, yet. With what you describe a
```
CommandLine|contains: ' -k '
```
would do just the same.

`|re` comes with increased cost (of rules in the matching engine, but also reduces compatibility. And I don't get the reason for using it, yet. I wouldn't suspect tabs or other whitespace being used in that position).

Reply:
The -k is case sensitive. Capital -K means config

Reply:
Then I also don't care about `\s` or ` `.